src/postgraphql/__tests__/__snapshots__/postgraphqlIntegrationSchema-test.js.snap

```diff
@@ -752,11 +736,14 @@ input PersonPatch {
   rowId: Int
 }
 
-type Post {
+type Post implements Node {
   authorId: Int
   body: String
   headline: String!
-  id: Int!
+
+  # A globally unique identifier. Can be used in various places throughout the system to identify this single value.
+  id: ID!
+  rowId: Int!
```

Comment:
Hmmm.... No this looks wrong. Previously it was:

https://github.com/postgraphql/postgraphql/blob/5cbc0bdcc904e9b37668f30a012a9f3e031a449c/src/postgraphql/__tests__/__snapshots__/postgraphqlIntegrationSchema-test.js.snap#L755-L760

There was no `GraphQLID` (`ID!`) field... Why?